### ISSUE TITLE
gestion des collections depuis l'espace privé

### ISSUE THREAD

**Issue by ubiquando** [geodiversite/geodiversite_albums#8]
Il n'y a pas d'accès aux collections dans l'espace ecrire.
Ce serait pas mal, non ?

**Comment by brunob** [comment]
Non, je pense que ça suffit amplement de pouvoir les gérer depuis l'espace public. Perso, je ne m'en occuperai pas, mais si quelqu'un d'autres veut coder la fonctionnalité, gogogo :)

**Comment by ubiquando** [comment]
ok, on va voir... ce serait aussi l'occasion de regarder les collections dynamiques, soit depuis une recherche, soit depuis un kml .
PAs gagné pour tout de suite...

**Comment by brunob** [comment]
Heu je reviens là dessus, mais en fait c'est déjà en place cf le lien dans les boutons d'admins depuis une page publique de collection qui renvoie bien vers `ecrire/?exec=collection&id_collection=X` et `ecrire/?exec=collections` qui liste bien toutes les collections du site. À la limite, il manque un lien vers la page des collections dans le menu Édition et ça c'est facile à corriger.

PS : je déplace le ticket sur le bon repo.
